front-end/src/components/FileButton.js

```diff
@@ -29,9 +47,17 @@ var ButtonStyle = {
         keepMounted
         open={Boolean(anchorEl)}
         onClose={handleClose}
-	anchorPosition={{ vertical: "top", horizontal: "left" }}
+        anchorPosition={{ vertical: "top", horizontal: "left" }}
       >
-        <MenuItem onClick={handleClose}>Import</MenuItem>
+        <MenuItem for="file-upload">
+          <label for="avatar">Import </label>
```

Comment:
in order to hide the Choose File button and only show Import.

`<label className="custom-file-upload">
          <input type="file" id="video-input" hidden
            accept="video/mp4, video/webm"
            onChange={handleImport} />
          <i className="fa fa-cloud-upload" /> Import
        </label>`

Reply:
I used your exact code to hide the choose file button now, thank you.